src/components/LandingPage/Header/index.jsx

```
@@ -41,6 +45,15 @@ const Header = () => {
         className={`header ${isVisible ? '' : 'header--hidden'}`}
         id="header"
       >
+        {announcement.show && (
+          <div className="header__announcement">
+            <div
+              className="header__announcement__wrapper"
+              dangerouslySetInnerHTML={{ __html: announcement.message }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)